hive/datadog_checks/hive/data/conf.yaml.example

```diff
@@ -1,848 +1,147 @@
+## All options defined here are available to all instances.
+#
+init_config:
+
+    ## @param is_jmx - boolean - required
+    ## Whether or not this file is a configuration for a JMX integration.
+    #
+    is_jmx: true
+
+    ## @param collect_default_metrics - boolean - required
+    ## Whether or not the check should collect all default metrics.
+    #
+    collect_default_metrics: true
+
+    ## @param conf - list of mappings - optional
```

Comment:
About JMX, `init_conf.conf` was required, but since we now use `metrics.yaml`, it shouldn’t be required anymore right?

Reply:
right